homeassistant/components/fastdotcom/sensor.py

```diff
@@ -24,11 +25,14 @@ async def async_setup_entry(
     async_add_entities: AddEntitiesCallback,
 ) -> None:
     """Set up the Fast.com sensor."""
-    async_add_entities([SpeedtestSensor(entry.entry_id, hass.data[DOMAIN])])
+    coordinator: FastdotcomDataUpdateCoordindator = hass.data[DOMAIN][entry.entry_id]
+    async_add_entities([SpeedtestSensor(entry.entry_id, coordinator)])
 
 
 # pylint: disable-next=hass-invalid-inheritance # needs fixing
```

Comment:
Should be able to remove this?

Reply:
Care to elaborate? Not sure that I understand why this isn't needed in the sensor setup.